bcipy/helpers/data_vizualization.py

```diff
@@ -19,7 +19,8 @@ def generate_offline_analysis_screen(
         fs=300,
         plot_x_ticks=8,
         plot_average=False,
-        show_figure=False) -> None:
+        show_figure=False,
+        channel_names = None) -> None:
```

Comment:
You could default to an empty dict here

Reply:
Yeah, I was avoiding a pylint dangerous-default-value warning, but in this case the channel_names dict is not mutated so a default arg would be fine.